src/databricks/labs/ucx/account/aggregate.py

```diff
@@ -75,12 +75,13 @@ def _federated_ucx_query(self, query: str, table_name='objects') -> Iterable[tup
                         TableMigrationStatus(
                             src_schema=inventory_database,
                             src_table=table_name,
-                            dst_catalog='hive_metastore',
+                            dst_catalog=ctx.config.inventory_catalog,
```

Comment:
revert, catalog will always be the same for this one. let's defer it to the time when we actually have our catalog.